pillar/environment-preview-public.sls

```diff
@@ -4,3 +4,21 @@ elife:
 journal:
     feature_xpub: false
     submit_url: https://reviewer.elifesciences.org/login
+    calls_to_action:
+      - id: eisen2019
+        path: ^(/$|/about$|/about/|/for-the-press$|/for-the-press/5cc21558/)
+        time:
+        text: 'Hear more about Michael Eisen, our new Editor-in-Chief, and the latest developments at eLife'
+        button:
+            text: 'Sign up for eLife news'
+            path: https://crm.elifesciences.org/crm/elife-news
+        image:
+            alt: ''
+            uri: https://iiif.elifesciences.org/journal-cms/person%%2F2019-03%%2Fmike.jpg
```

Comment:
`Bad request`. Presumably because it's not live yet?

Reply:
There's a double `%` (escaping for Symfony, change it to one and it should work).